static/app/views/replays/detail/network/index.tsx

```diff
@@ -216,9 +217,11 @@ function NetworkList({replayRecord, networkSpans}: Props) {
           {`${(networkEndTimestamp - networkStartTimestamp).toFixed(2)}ms`}
         </Item>
         <Item {...columnHandlers} {...columnProps} numeric>
-          <UnstyledButton onClick={() => handleClick(networkStartTimestamp)}>
-            {showPlayerTime(networkStartTimestamp, startTimestampMs, true)}
-          </UnstyledButton>
+          <Tooltip title={<DateTime date={networkStartTimestamp} seconds />}>
+            <UnstyledButton onClick={() => handleClick(networkStartTimestamp)}>
+              {showPlayerTime(networkStartTimestamp, startTimestampMs, true)}
+            </UnstyledButton>
+          </Tooltip>
```

Comment:
Wonder if this should be a common component

Reply:
it should, it's related to #39299 as well.